src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -61,7 +61,7 @@ const AnnotationActivity = ({
         : { ...linkMessage, values: { number: linkValue } };
 
     return (
-        <div className="bcs-AnnotationActivity">
+        <div className="bcs-AnnotationActivity" data-resin-feature="annotations">
```

Comment:
This is not annotations, though, it's activity feed.

Reply:
All of the annotations related resin tags have `feature="annotations"` I think it's just easy for Rohit to collect annotations logs.